web/templates/left_sidebar.hbs

```diff
@@ -1,8 +1,10 @@
 <div class="left-sidebar" id="left-sidebar" role="navigation">
-    <div id="left-sidebar-search" class="zoom-in-hide tippy-zulip-delayed-tooltip" data-tooltip-template-id="filter-left-sidebar-tooltip-template">
-        {{#> input_wrapper input_type="filter-input" custom_classes="left-sidebar-search-section" icon="search" input_button_icon="close"}}
-            <input type="text" class="input-element left-sidebar-search-input home-page-input" autocomplete="off" placeholder="{{t 'Filter left sidebar' }}" />
-        {{/input_wrapper}}
+    <div id="left-sidebar-search" class="zoom-in-hide">
+        <div class="input-wrapper-for-tooltip tippy-zulip-delayed-tooltip" data-tooltip-template-id="filter-left-sidebar-tooltip-template">
+            {{#> input_wrapper input_type="filter-input" custom_classes="left-sidebar-search-section" icon="search" input_button_icon="close"}}
+                <input type="text" class="input-element left-sidebar-search-input home-page-input" autocomplete="off" placeholder="{{t 'Filter left sidebar' }}" />
+            {{/input_wrapper}}
+        </div>
```

Comment:
This seems fine, but I think probably the `input_wrapper`  itself could have these classes; we'd just need a way to pass a `data-tooltip-template-id` to it.